payment-failure/src/main/scala/com/gu/identity/paymentfailure/SendEmailService.scala

```diff
@@ -5,7 +5,7 @@ class SendEmailService (identityClient: IdentityClient, brazeClient: BrazeClient
   def sendEmail(emailData: IdentityBrazeEmailData, config: Config): Either[Throwable, BrazeResponse] = {
     for {
       encryptedTokenResponse <- identityClient.encryptEmail(emailData.emailAddress, config)
-      brazeResponse <- brazeClient.sendEmail(emailData, encryptedTokenResponse.encryptedEmail)
+      brazeResponse <- brazeClient.sendEmail(emailData, encryptedTokenResponse.encryptedEmail, config)
```

Comment:
Hi-jacking this PR with a comment not related to it - realised that config it getting passed to `encryptEmail()`, but it doesn't need it since config is getting injected as a class constructor to the identity client.

Reply:
Yep good point thanks